project-fortis-pipeline/localdeploy/parameters.json

```diff
@@ -3,28 +3,28 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "namespaces_fortiseventhubs_name": {
-            "value": "fortisev"
+            "value": "rachelfortisev"
```

Comment:
Please don't check in changes to this file since it's being used as a template for the local Azure resources setup script.

Reply:
my mistake! sorry about that.